packages/uicore/src/components/MultiTypeInput/LearnMorePopover.tsx

```diff
@@ -35,6 +35,11 @@ const helperText: Record<MultiTypeInputType, React.ReactNode> = {
       <b>Runtime Inputs</b> are placeholders for values that will be provided when you start a Pipeline execution.
     </React.Fragment>
   ),
+  [MultiTypeInputType.RUNTIMEY1]: (
+    <React.Fragment>
+      <b>Runtime Inputs</b> are placeholders for values that will be provided when you start a Pipeline execution.
```

Comment:
This text might change now, can we once confirm with product on this

Reply:
This holds for new inputs. I've asked the product about this. I'll update if they say otherwise.